Very Boring low impact changes

SpicyAlfredo opened the pull request.

SpicyAlfredo committed:
Update LUX_l_english.yml

SpicyAlfredo committed:
Update r56_SOV_l_english.yml

SpicyAlfredo committed:
Update r56_sinkiang.txt

SpicyAlfredo committed:
Update r56_generic_tree.txt

marijn211 commented:
Punctuation seems off in the Luxembourgh file edit, split sentence. r56.soviet.7.a is now wrong
(some of the changed soviet strings also sounded funnier before but that's not top priority)

SpicyAlfredo committed:
asfd

SpicyAlfredo commented:
> Punctuation seems off in the Luxembourg file edit, split sentence. r56.soviet.7.a is now wrong (some of the changed soviet strings also sounded funnier before but that's not a top priority)

Fixed  r56.soviet.7.a.

SpicyAlfredo committed:
Went back to less bad run on sentence

ilkober committed:
Merge branch 'Beta-Edits' into SpicyAlfredo-patch-3

ilkober committed:
Update r56_SOV_l_english.yml

ilkober reviewed (approved):
Looks good
